src/Apps/Settings/SettingsApp.tsx

```diff
@@ -44,48 +41,20 @@ const SettingsApp: React.FC<SettingsAppProps> = ({ me, children }) => {
     })
   }, [location.query.error, sendToast])
 
-  const tabsWhenCollectorProfileEnabled = compact([
-    { name: "Edit Profile", url: "/settings/edit-profile" },
-    { name: "Saved Alerts", url: "/settings/alerts" },
-    { name: "Account", url: "/settings/edit-settings" },
-    { name: "Order History", url: "/settings/purchases" },
-    { name: "Bids", url: "/settings/auctions" },
-    { name: "Payments", url: "/settings/payments" },
-    { name: "Shipping", url: "/settings/shipping" },
-  ])
-
-  if (!isLoggedIn) {
-    return <MyCollectionRouteLoggedOutState />
-  }
-
   return (
     <>
       <MetaTags title="Settings | Artsy" />
 
-      <>
-        <Breadcrumbs py={[2, 1]}>
-          <Link to="/collector-profile/my-collection">
-            <Flex flexDirection="row" alignItems="center" minHeight={30}>
-              <ChevronIcon
-                direction="left"
-                color="black100"
-                height={14}
-                width={14}
-              />
-              <Text variant="xs" pl={1}>
-                Collector Profile
-              </Text>
-            </Flex>
-          </Link>
-        </Breadcrumbs>
+      <TopContextBar displayBackArrow href="/collector-profile/my-collection">
+        Collector Profile
+      </TopContextBar>
 
-        <Spacer y={4} />
-      </>
+      <Spacer y={[4, 6]} />
 
       <RouteTabs my={SETTINGS_ROUTE_TABS_MARGIN}>
-        {tabsWhenCollectorProfileEnabled.map(tab => {
+        {SETTINGS_TABS.map(tab => {
           return (
-            <RouteTab key={tab.url} to={tab.url} variant={["xs", "sm"]}>
```

Comment:
Avoid overwriting styles coming from the design system. On mobile you want tap areas to be bigger, really, anyway.